src/components/BaseComponents/CheckAnswer/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from 'react';
+import type { PConnFieldProps } from '@pega/react-sdk-components/lib/types/PConnProps';
+import { useTranslation } from 'react-i18next';
+
+interface HmrcOdxTestProps extends PConnFieldProps {
+  // If any, enter additional props that only exist on this componentName
+  name?: string;
+  stepId?: any;
+}
+
+// Duplicated runtime code from React SDK
+
+// props passed in combination of props from property panel (config.json) and run time props from Constellation
+// any default values in config.pros should be set in defaultProps at bottom of this file
+export default function GDSCheckAnswers(props: HmrcOdxTestProps) {
+  const COMMA_DELIMITED_FIELD = 'CSV';
+  const { label, value, name, stepId, getPConnect } = props;
+  const [formattedValue, setFormattedValue] = useState<string | Array<string>>(value);
+  const pConn = getPConnect();
+  const actions = pConn.getActionsApi();
+  const containerItemID = pConn.getContextName();
+  const { t } = useTranslation();
+  let isCSV = false;
+
+  if (name && name.includes(COMMA_DELIMITED_FIELD) && value.includes(',')) {
+    isCSV = true;
+  }
+
+  useEffect(() => {
+    if (name && name.includes(COMMA_DELIMITED_FIELD) && value.includes(',')) {
+      const formatValue = value.split(',').map((item: string) => item.trim());
+      setFormattedValue(formatValue);
+    }
+  }, []);
+  const handleOnClick = () => {
+    const navigateToStepPromise = actions.navigateToStep(stepId, containerItemID);
```

Comment:
the code can be changed here to `actions.navigateToStep(stepId, containerItemID)` and removing the const navigateToStepPromise  for readability.

Reply:
You dont need to do now but as we discussed in the next code update as this is urgent It can go